code/style.css

```diff
@@ -1,7 +1,500 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+ body {
+   margin: 0;
+   font-family: 'Source Sans Pro', sans-serif;
+   background-color: white;
+   color: #333;
+ }
 
-.active + .answer {
-  /* some styling here */
-}
+ a {
+   color: #333;
+ }
+
+ /***************** Hero Video ****************/
+
+ .hero-wrapper-container {
+   display: flex;
+   justify-content: center;
+   justify-self: stretch;
+   align-items: flex-start;
+ }
+
+ .hero-styling {
+   width: 100%;
+   height: 100%;
+   position: relative;
+   object-fit: cover;
+   overflow: hidden;
+   left: 0;
+   right: 0;
+   top: 0;
+   bottom: 0;
```

Comment:
This I have never seen, what does it do?

Reply:
this moves an element in relation to the view port (if position of element is fixed ) or in relation to the closest parent element with position relative .

Reply:
it will work only along with position fixed/absolut